Readme.md

```diff
@@ -64,9 +64,12 @@ If you do not add credentials to your profile, a credential prompt displays when
     - **True**: Reject connections with self-signed certificates.
     - **False**: Accept connections with self-signed certificates.
     
-Your profile is now available in the panel on the left. You can also use this profile in Zowe CLI directly.
-
-If you have multiple profiles, you might wish to delete some once you no longer need them. To do so, click on the trash can icon on the same line as the profile name.
```

Comment:
Also, the tool tip on the trash can states `Remove Profile`, as not to confuse the users that the profile is deleted from the system as well. Please see:
https://github.com/eclipse/che-che4z-explorer-for-endevor/blob/c9ae4f159730435e70c7c6a4da74d90cd27b2ca0/package.json#L60-L61

Reply:
I've changed the wording to better reflect this